spec/lib/embed/viewer/common_viewer_spec.rb

```diff
@@ -104,6 +104,22 @@
       html = Capybara.string(file_viewer.to_html)
       expect(html).to have_css '.sul-embed-container[style="display:none; max-height:200px; max-width:200px;"]', visible: false
     end
+
+    context 'fullheight' do
+      it 'includes the sul-embed-fullheight class if the fullheight option is passed' do
+        stub_purl_response_and_request(file_purl, request)
+        expect(request).to receive(:fullheight?).at_least(:once).and_return(true)
+        html = Capybara.string(file_viewer.to_html)
+        expect(html).to have_css('.sul-embed-fullheight', visible: false)
```

Comment:
The viewer is `display:none` by default and is displayed via JS once the JS and CSS is loaded.

Since we're using `Capybara.string` above we're really just using the capybara DSL to make assertions against the HTML (not executing any javascript).  That means that the element we're checking is `display:none` so we have to say `visible: false` otherwise Capybara will say the element does not exist (which it does).